### PR TITLE
Fixed possible memory leak

### PR DESCRIPTION
**Pull request by Goblenus**

### CODE REVIEW

File: crypto/LPdir_win.c [hunk 1]
```diff
@@ -104,6 +104,9 @@ const char *LP_find_file(LP_DIR_CTX **ctx, const char *directory)
              * the ending NUL.
              */
             if ((len_0 - 1) > MAX_PATH * sizeof(TCHAR)) {
+                if (extdirbuf != NULL) {
```
Comment (richsalz) [inline crypto/LPdir_win.c:107]: you don't need the if check; `free(NULL);` works fine.

Comment (paulidale) [reply crypto/LPdir_win.c:107]: Agreed.

You'll also need to add "CLA: trivial" on a line by itself in the commit message (use git commit --amend) or submit a signed ICLA.

Comment (Goblenus) [reply crypto/LPdir_win.c]: Done. Check please.

**Review by paulidale — APPROVED** [review]

File: crypto/LPdir_win.c (hunk 1 above)
Comment (paulidale) [reply crypto/LPdir_win.c]: Thanks.

**Comment by paulidale** [comment]
I just realised that this is for 1.0.2 which is long unsupported.  It doesn't look like it is applicable to 1.1.1 or master.

**Comment by Goblenus** [comment]
Yep. But we still using 1.0.2 and I decided to do this PR. Any chances to merge it to 1.0.2?

**Comment by paulidale** [comment]
I think it very unlikely.

**Pull request closed without merging**